examples/wdio/mocha/mochaWithDataProvider.test.js

```diff
@@ -0,0 +1,10 @@
+const assert = require('assert')
+
+describe(`${testData.url} page`, () => {
```

Comment:
can you create a independent wdio example for data provider? The current mocha example should stay as simple mocha example.

Reply:
@christian-bromann : I have created a new spec file ```mochaWithDataProvider.test.js``` for mocha with data provider example.

Reply:
@yamkay can you create a whole different example with data providers (e.g. in `/examples/wdio/dataProviders`)